### ISSUE TITLE
[Minor] Please don't ask to accept the GPLv2

### ISSUE THREAD

**Issue by Flameeyes** [Spagett1/PineFlash#60]
GPL is not an EULA, there's no need to accept it.

See https://flameeyes.blog/2020/03/23/the-gpl-is-not-an-eula/ for the long (ranty) version of this request :)

**Comment by Spagett1** [comment]
Yeah that article has a fair point, I'll edit it when I get a chance and next release shouldn't have it.

**Comment by Spagett1** [comment]
Thanks for bringing it to my attention

**Comment by acuteaangle** [comment]
This only occurs on Windows. This is still applicable to 0.5.5.
